File: Front End/src/app/Shared/Model/Passenger.ts
export class Passenger {
  PassengerAge: number;
  PassengerName: string;
  FlightId: number;
  public constructor(PassengerName, PassengerAge, FlightId) {
    this.PassengerName = PassengerName;
    this.FlightId = FlightId;
    this.PassengerAge = PassengerAge;
  }
}

File: Backend/FlightManagement/FlightManagement/Domain/Entities/Passenger.cs
using System;
using System.Collections.Generic;

namespace FinalCodingChallenge.Domain.Entities
{
    public partial class Passenger
    {
        public Passenger(string passengerName, int? passengerAge, int? flightId)
        {
            PassengerName = passengerName;
            PassengerAge = passengerAge;
            FlightId = flightId;
        }

        public int PassengerId { get; set; }
        public string PassengerName { get; set; }
        public int? PassengerAge { get; set; }
        public int? FlightId { get; set; }

        public Flight Flight { get; set; }
    }
}
